File: result/translate/starchat/java2cpp/c++/CALCULATE_MAXIMUM_VALUE_USING_SIGN_TWO_NUMBERS_STRING.cpp
int f_gold ( string str ) {
    int res = (int)str[0] - '0';
    for ( int i = 1; i < str.length(); i++ ) {
        if ( str[i] == '0' || str[i] == '1' || res < 2 ) res += (int)str[i] - '0';
        else res = res * 10 + (int)str[i] - '0';
    }
    return res;
}

File: result/translate/chatgpt/java2python/python/CALCULATE_MAXIMUM_VALUE_USING_SIGN_TWO_NUMBERS_STRING.py
def f_gold(str):
    res = ord(str[0]) - ord('0')
    for i in range(1, len(str)):
        if str[i] == '0' or str[i] == '1' or res < 2:
            res += (ord(str[i]) - ord('0'))
        else:
            res += (ord(str[i]) - ord('0'))
    return res